src/app/components/InvoiceForm/index.tsx

```diff
@@ -241,21 +247,26 @@ class InvoiceForm extends React.Component<Props, State> {
   };
 
   private updateBothValues = (name: string, val: string) => {
-    const { fiat, rates } = this.props;
+    const { fiat, rates, nodeInfo } = this.props;
     const { denomination } = this.state;
     let { value, valueFiat } = this.state;
+
+    let rateSelector = 'btcRate';
```

Comment:
it will be simpler to just key off of the chain name 'bitcoin' and 'litecoin' instead of using a new key 'btcRate' and 'ltcRate'